service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureControlledStorageResourceService.java

```diff
@@ -34,6 +34,8 @@
 @Component
 public class AzureControlledStorageResourceService {
 
+  public record AzureSasBundle(String sasToken, String sasUrl) {}
```

Comment:
This is my solution to the "I want to return two values from a method" problem, rather than a `Pair<String, String>`. Open to other suggestions.